torch/_torch_docs.py

```diff
@@ -8036,6 +8036,18 @@ def merge_dicts(*dicts):
     possible to satisfy the definition of a modulo operation with complex numbers.
     See :func:`torch.fmod` for how division by zero is handled.
 
+.. note::
+    This is different from `math.remainder <https://docs.python.org/dev/library/math.html#math.remainder>`_
```

Comment:
Slight grammar tweak:

"This op, like NumPy's remainder, is equivalent to Python's modulus operation, and different from Python's `math.remainder` and C++'s `std::remainder` which implement the IEEE remainder.
"

The references for both Python and C++ are great so let's keep them, too

The second paragraph is interesting but I think this first paragraph tells readers everything they need to know, so we can remove the second paragraph

Reply:
Sounds good, @mruberry - Thanks! I've pushed the changes, I'll wait for the CI to build the docs and check the rendered link to just be sure.

Once I think it's good to go, I'll ping you here. Thank you!

P.S: I added NumPy's remainder reference as well, hope that sounds fine?